mlflow/tracing/types/wrapper.py

```diff
@@ -187,18 +193,16 @@ def to_mlflow_span(self):
         return Span(
             name=self._span.name,
             context=SpanContext(
-                request_id=self.request_id,
+                trace_id=self._span.get_span_context().trace_id,
```

Comment:
Can we make trace_id a property as well?

Reply:
No, we don't want to expose it to users, because it will be the source of mistake e.g. specify trace ID for client APIs.